Description: GSSAPI configuration dump fixes
 Add GSSAPIKeyExchange and GSSAPIStrictAcceptorCheck to sshd -T
 configuration dump.
Author: Colin Watson <cjwatson@debian.org>
Last-Updated: 2010-02-27

Index: b/servconf.c
===================================================================
--- a/servconf.c
+++ b/servconf.c
@@ -1624,7 +1624,9 @@
 #endif
 #ifdef GSSAPI
 	dump_cfg_fmtint(sGssAuthentication, o->gss_authentication);
+	dump_cfg_fmtint(sGssKeyEx, o->gss_keyex);
 	dump_cfg_fmtint(sGssCleanupCreds, o->gss_cleanup_creds);
+	dump_cfg_fmtint(sGssStrictAcceptor, o->gss_strict_acceptor);
 #endif
 #ifdef JPAKE
 	dump_cfg_fmtint(sZeroKnowledgePasswordAuthentication,
